### PR TITLE
 add Win 64bit to spansh download-easy (#28288)

### PR DESCRIPTION
**Pull request by traumschule**
https://bugs.torproject.org/28288

### CODE REVIEW

File: download/es/download-easy.wml [hunk 1]
```diff
@@ -52,7 +75,7 @@
 	      (<a class="win-tbb-sig" href="../dist/torbrowser/<version-torbrowserbundle>/torbrowser-install-<version-torbrowserbundle>_<lang>.exe.asc">firma</a>) <a class="siginfo" href="<page docs/verifying-signatures>">¿Que es Esto?</a>
 	    </div>
 	  </form>
-	  <p class="alt-dl">¿No usas Windows?<br />Descargalo para <a href="#mac">Mac</a> o <a href="#linux">Linux</a></p>
+	  <p class="alt-dl">¿No usas Windows?<br />Descargalo para <a href="#mac">Mac</a> or <a href="#linux">Linux</a></p>
```
Comment (gusgustavo) [inline download/es/download-easy.wml]: Please, change this to "o" ("or" in spanish)

Comment (traumschule) [reply download/es/download-easy.wml]: done

File: download/es/download-easy.wml [hunk 2]
```diff
@@ -19,105 +19,184 @@
...
+          <p>
+          Known issue: Our Security Slider is now under ‘Security Settings,
+          but because of a small issue, it’s only showing up after you restart
+          the app. We plan on fixing it for the next release.
+          </p>
+          <form class="downloads">
+            <a class="button" href="../dist/torbrowser/mobile/<version-torbrowserbundleandroid>/tor-browser-android-arm-<version-torbrowserbundleandroid>.apk"><span class="strong">Download</span><span class="normal">Android</span></a>
+            <div class="sig">
+              (<a class="android-tbb-sig" href="../dist/torbrowser/mobile/<version-torbrowserbundleandroid>/TorBrowser-<version-torbrowserbundleandroid>.asc">sig</a>)
+              <a class="siginfo" href="<page docs/verifying-signatures>">What's This?</a>
```
Comment (traumschule) [inline download/es/download-easy.wml]: This part has been updated by 73d98b306 and needs to be rebased and translated.

Are we ok to uncomment the android section (also on the english download-easy page)?

**Comment by traumschule** [comment]
merged

**Pull request closed without merging**